cookbook/entities/managing_tasks.rst

```diff
@@ -0,0 +1,254 @@
+How to Create and Modify Tasks
+==============================
+
+To let the users create new tasks and edit existing ones you need to perform several steps:
+
+#. :ref:`Create a form type for the Task entity <cookbook-entity-form-type>`.
+
+#. :ref:`Create a controller <cookbook-entity-controller>`.
+
+#. :ref:`Render the form in a template <cookbook-entity-template>`.
+
+#. :ref:`Link data grid entries to the controller <controller-entity-grid-create-edit-link>`.
+
+.. _cookbook-entity-form-type:
+
+The Form Type
+-------------
+
+First, you need to create a form type that makes it possible to let the user enter all the data
+needed to describe a task:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/Form/TaskType.php
+    namespace AppBundle\Form;
+
+    use Symfony\Component\Form\AbstractType;
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\OptionsResolver\OptionsResolver;
+
+    class TaskType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder)
+        {
+            $builder
+                ->add('subject')
+                ->add('description')
+                ->add('dueDate')
+                ->add('priority')
+            ;
+        }
+
+        public function configureOptions(OptionsResolver $resolver)
+        {
+            $resolver->setDefaults(array(
+                'data_class' => 'AppBundle\Entity\Task',
+            ));
+        }
+    }
+
```

Comment:
Please add `More info about Symfony forms you can find here http://symfony.com/doc/current/book/forms.html`